platform/admin/ui/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -39,17 +39,6 @@
                 required="true" type="Boolean"
                 default="true"
         />
-
-        <AD
-                description="Specifies the header text to be rendered on the Administrator Console."
-                name="Header" id="header" required="true" type="String"
-                default=""
-                />
-        <AD
-                description="Specifies the footer text to be rendered on the Administrator Console."
-                name="Footer" id="footer" required="true" type="String"
-                default=""
-                />
```

Comment:
✅ These are under `platform/admin` and I believe are the correct configs to remove. They weren't serving any value. A hero of this PR should verify banners can still be configured, **and** that any custom guest claims profiles are still executed by the installer correctly.